cmd/driver/install/install.go

```diff
@@ -172,30 +174,88 @@ func (o *driverInstallOptions) RunDriverInstall(ctx context.Context, driver *con
 	}
 	o.Printer.Logger.Info("found distro", o.Printer.Logger.Args("target", d))
 
-	err = driver.Type.Cleanup(o.Printer, driver.Name)
+	var (
+		dest string
+		buf  bytes.Buffer
+	)
+
+	if !o.Printer.DisableStyling {
+		o.Printer.Spinner, _ = o.Printer.Spinner.Start("Cleaning up existing drivers")
+	}
+	err = driver.Type.Cleanup(o.Printer.WithWriter(&buf), driver.Name)
+	if o.Printer.Spinner != nil {
+		_ = o.Printer.Spinner.Stop()
+	}
+	if o.Printer.Logger.Formatter == pterm.LogFormatterJSON {
```

Comment:
This is ugly: we should probably find a better way to deal with unformatted logs while in JSON mode.